homeassistant/data_entry_flow.py

```diff
@@ -49,17 +49,25 @@ def __init__(self, reason: str, description_placeholders: Optional[Dict] = None)
 class FlowManager:
     """Manage all the flows that are in progress."""
 
-    def __init__(
-        self,
-        hass: HomeAssistant,
-        async_create_flow: Callable,
-        async_finish_flow: Callable,
-    ) -> None:
+    def __init__(self, hass: HomeAssistant,) -> None:
         """Initialize the flow manager."""
         self.hass = hass
         self._progress: Dict[str, Any] = {}
-        self._async_create_flow = async_create_flow
-        self._async_finish_flow = async_finish_flow
+
+    async def async_create_flow(
```

Comment:
Mark these with `@abstractmethod` (imported from `abc`). The `raise` body won't be necessary anymore.